### PR TITLE
Documentation for num_columns_recent + using site

### PR DESCRIPTION
**Pull request by michelle192837**

### CODE REVIEW

File: testgrid/config/README.md [hunk 1]
```diff
@@ -164,7 +172,79 @@ test_groups:  # Attach to a specific test_group
...
+Here are some quick tips and clarifications for using the TestGrid site!
+
+### Customizing Test Result Sizes
+
+Change the size of the test result rectangles.
+
+The three sizes are Standard, Compact, and Super Compact. You can also specify
+`width=X` in the URL (X > 3) to customize the width. For small widths, this may
+mean the date and/or changelist, or other custom headers, are no longer
+visible.
```
Comment (krzyzacy) [inline testgrid/config/README.md]: maybe a screenshot can be more intuitive

**Comment by krzyzacy** [comment]
/lgtm

I'll get this in and let you make follow up PRs :-)

File: testgrid/config/README.md (hunk 1 above)
Comment (michelle192837) [inline testgrid/config/README.md]: Noted; following up more thoroughly with screenshots for this later, filed #4040

**Pull request merged**